### PR TITLE
Adding code owners

### PR DESCRIPTION
**Pull request by TheKnarf**

### CODE REVIEW

**Review by Trulsaa — APPROVED** [review]

**Comment by TheKnarf** [comment]
What's going on with the build? It doesn't seem to be starting.

**edit**
I think I know what's going on.
This line: https://github.com/Alv-no/alvtime-monorepo/blob/master/.github/workflows/docker-push-dev.yml#L10
It ignores CODEOWNER files, meaning no build will be triggered.
But in the settings I have setup that the build must pass before you can merge changes inn.

So it's deadlocked, no build will be triggered, but without a build it won't let us merge.

**Pull request merged**
